picard/ui/metadatabox/__init__.py

```diff
@@ -335,15 +347,15 @@ def _copy_value(self):
             # Just copy the current item as a string
             item = self.currentItem()
             if item:
-                tag, value = self.get_row_info(item.row())
+                tag, value = self._get_row_info(item.row())
                 value = value[item.column()]
                 if tag == '~length':
                     value = self.tag_diff.handle_length(value, prettify_times=True)
                 if value is not None:
                     log.debug("Copying '%s' to clipboard (from tag '%s')", value, tag)
                     self.tagger.clipboard().setText(MULTI_VALUED_JOINER.join(value))
 
-    def _paste_multiple(self, data):
+    def _paste_from_json_string(self, data):
```

Comment:
Data isn't exactly a json string, but a dict, not sure the function name is clear enough.

Reply:
Valid